docs/source/index.rst

```diff
@@ -3,8 +3,11 @@
 Conda Build Documentation
 =========================
 
-Building a conda package requires
-:doc:`installing conda-build <install-conda-build>` and
+Conda build contains commands and tools to allow
+you to build your own packages for conda.
```

Comment:
Consider rewording slightly to "...build your own conda packages" or "use conda to build your own packages"